libs/exavmlib/lib/List.Chars.Float.ex

```diff
@@ -0,0 +1,26 @@
+#
+# This file is part of elixir-lang.
+#
+# Copyright 2013-2023 Elixir Contributors
+# https://github.com/elixir-lang/elixir/commits/v1.17.2/lib/elixir/lib/list/chars.ex
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+defimpl List.Chars, for: Float do
+  def to_charlist(term) do
+    :erlang.float_to_list(term, [:short])
```

Comment:
This will fail, in `String.Chars.Float` I did this:
```elixir
 26     # TODO: :short option not yet supported right now, so :decimals+:compact should be replaced
 27     :erlang.float_to_binary(term, [{:decimals, 17}, :compact])
 ```